templates/fragments/_sca.erb

```diff
@@ -1,70 +1,39 @@
 <%- if @kernel == 'Linux' -%>
-  <%- if @apply_template_os == 'centos' -%>
     <sca>
       <enabled>yes</enabled>
       <scan_on_start>yes</scan_on_start>
       <interval>12h</interval>
       <skip_nfs>yes</skip_nfs>
 
       <policies>
+      <%- if @os['family'] == 'CentOS' -%>
         <policy>cis_rhel7_linux_rcl.yml</policy>
+      <%- elsif @os['family'] == 'Amazon' -%> 
```

Comment:
empty if body, remove this line

Reply:
The empty body is used to disable EL rules.

Amazon Linux matches with family RedHat, but the EL rules are not used upstream, so I've disabled them here as well.

https://github.com/wazuh/wazuh-puppet/blob/master/templates/fragments/_sca.erb#L16